README.md

```diff
@@ -1,2 +1,139 @@
-# flat-image-tree
-Spec for the Flat Image Tree (FIT) specification
+# Document master project for the Devicetree Specification #
+
+The latest release of the specification can be found at http://devicetree.org/specifications/ or https://github.com/devicetree-org/devicetree-specification/releases
+
+This [repository](https://github.com/devicetree-org/devicetree-specification) holds the source for the generation of the Devicetree Specification using Sphinx and LaTeX. 
+
+## Mailing list: devicetree-spec@vger.kernel.org ##
+   * Use this mailing list for submitting patches, questions and general discussion
+   * Sign up to the mailing list at http://vger.kernel.org/vger-lists.html#devicetree-spec
+
+## Build Instructions [![ci](https://github.com/devicetree-org/devicetree-specification/actions/workflows/ci.yml/badge.svg)](https://github.com/devicetree-org/devicetree-specification/actions/workflows/ci.yml) ##
```

Comment:
This part still refers to DT spec. Is this intended for now and will be updated later?

Reply:
fixed